source/NationalInstruments/jquery.flot.historybuffer.analogWaveform.js

```diff
@@ -96,14 +96,14 @@ console.log(hb1.toDataSeries()); //[[4, 1], [5, 2], [6, 3], [null, null], [1, 1]
         var waveformEnd = t0.valueOf() + aw.Y.length * aw.dt;
 
         if (waveformStart < start && waveformEnd < start) {
-            return false;
+            return -1;
```

Comment:
Could this function return some sort of enum instead? Something like this:

```
const WaveformVisibility = Object.freeze({
  BeforeRange:1,
  InRange:2, 
  AfterRange:3
})
```

Reply:
The reason I went with the values I went with was simply because of standard return values for compare functions (-1 being less than, 0 being equivalent, and +1 being more than). While there isn't an exact match for this scenario, it's certainly similar. Do you still prefer the enum to that with that in mind (I could go either way)?

Reply:
Yeah I get what you're going for, I just found it hard to look at the code where this is being used and quickly understand what the values represent. I think I'd prefer the enum, especially since this function isn't used for a sort or anything that would require the int return value.

Reply:
Updated.